### DISCUSSION TITLE
Error saat ingin deploy ke shiny

### DISCUSSION THREAD

**Discussion by rizkiajimahardika** [teamalgoritma/community#101] [Q&A] [answered]
Error saat ingin deploy ke shiny
![error shiny](https://user-images.githubusercontent.com/79896604/168229072-44aad616-dbc2-4229-ba21-6fe528b338bd.png)

**Comment by inytss** [comment, marked as answer]
Dear @rizkiajimahardika ,
Mengenai error deploy sepertinya sedang terjadi error pada server shinyapps.io nya. Mohon dicoba lagi nanti. Sebelum melakukan deploy, pastikan Bapak sudah menghubungkan akun shinyapps Bapak dengan RStudio, selain itu data yang akan digunakan dalam pembuatan dashboard diikut sertakan saat proses deploy.
Semoga ketika mencoba lagi, tidak terdapat eror deploy kembali.

**Reply by rizkiajimahardika** [reply]
baik terima kasih bu atas jawabannya
